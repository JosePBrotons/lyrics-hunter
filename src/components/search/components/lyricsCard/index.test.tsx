import React from 'react';
import renderer from 'react-test-renderer';
import LyricsCard from '.';

test('renders lyrics card component correctly', () => {
    const props = {
        artist: 'Coldplay',
        lyrics: 'I used to rule the world Seas would rise when I gave the word',
        song: 'Viva La Vida',
        onPress: jest.fn
    };
    const lyricsCardCmp = renderer.create(<LyricsCard {...props} />).toJSON();
    expect(lyricsCardCmp).toMatchSnapshot();
});