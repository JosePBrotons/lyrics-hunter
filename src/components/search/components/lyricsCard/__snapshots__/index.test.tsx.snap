// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders lyrics card component correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "rgba(255, 255, 255, 1)",
      "borderRadius": 5,
      "flexDirection": "column",
      "marginVertical": 15,
      "opacity": 1,
      "padding": 16,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "rgba(69, 69, 69, 1)",
        "fontFamily": "Nunito-Bold",
        "fontSize": 16,
        "lineHeight": 22,
      }
    }
  >
    Viva La Vida
  </Text>
  <Text
    style={
      Object {
        "color": "rgba(69, 69, 69, 1)",
        "fontFamily": "Nunito-Bold",
        "fontSize": 14,
        "lineHeight": 22,
      }
    }
  >
    Coldplay
  </Text>
  <Text
    ellipsizeMode="tail"
    numberOfLines={1}
    style={
      Object {
        "color": "rgba(106, 106, 106, 1)",
        "fontFamily": "Nunito-Regular",
        "fontSize": 14,
        "lineHeight": 22,
      }
    }
  >
    I used to rule the world Seas ...
  </Text>
</View>
`;
